index.html

```diff
@@ -303,6 +301,50 @@ <h5 class="header-5">About Me</h5>
             </li>
         </ul>
     </section>
-</body>
-
+    <section id="contact-form">
+        <div id="header-6">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+        </div>
```

Comment:
- [ ] You did an amazing job add contact form section but kindly change it to footer tag as it was part of [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Keep rocking 💪